autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/daos/configurations/ProvideDataBaseProperties.java

```diff
@@ -35,6 +35,10 @@
 @Configuration
 public class ProvideDataBaseProperties {
 
+    /**
+     * Returns the database ".properties" files to be accessible to the Autonomiccs plugins

```

Comment:
It exports "database.properties" object to the Autonomiccs plugins application context